docs/extend/extend-cli/cli-developing-a-plugin.md

```diff
@@ -1,219 +1,348 @@
-# Developing a new plug-in
+# Developing a new Zowe CLI plug-in
 Before you begin this tutorial, complete the [Extending an existing plug-in](cli-extending-a-plugin.md) tutorial.
 
 ## Overview
-This tutorial demonstrates how to create a brand new Zowe&trade; CLI plug-in that uses Zowe CLI Node.js programmatic APIs.
+The advantage of Zowe CLI and of the CLI approach in mainframe development is that it allows for combining different developer tools for new and interesting uses.
```

Comment:
I agree w/ what you're saying, I just don't know if this is the article where this needs to be said. I can see this belong better in more general/overview articles about Zowe CLI. I'm stealing your words so that I can use them elsewhere!